### PR TITLE
[green-dragon-migration] Add lldb jobs

### PR DESCRIPTION
**Pull request by justice-adams-apple**
## In This PR
* Update all existing lldb jobs

All lldb jobs have been updated to:
* Run on the correct labels
* upload artifacts to S3
* Set environment variables in `environment` closure when possible
* Setup python3 venv with all dependencies to build/test

This code has been tested on https://green.lab.llvm.org/

### CODE REVIEW

**Comment by justice-adams-apple** [comment]
cc @azharudd

File: zorg/jenkins/jobs/jobs/lldb-cmake-standalone [hunk 1]
```diff
@@ -184,19 +230,12 @@ pipeline {
...
-                    ],
-                    replyTo: '$DEFAULT_REPLYTO',
-                    to: '$DEFAULT_RECIPIENTS',
-                    body:'$DEFAULT_CONTENT'
+        always {
+            // ToDo: Restore issue scanner
+            // scanForIssues tool: clang()
         }
     }
+    */
```
Comment (JDevlieghere) [inline zorg/jenkins/jobs/jobs/lldb-cmake-standalone:240]: For the standalone bot we can just get rid of this.

File: zorg/jenkins/jobs/jobs/lldb-cmake-intel [hunk 2]
```diff
@@ -35,23 +48,52 @@ pipeline {
...
+            }
+            steps {
+              timeout(90) {
+                  sh '''
+                  source ./venv/bin/activate
+
+                  # The relay job sets ARTIFACT to the last good LLDB build.
+                  # Here we use it to download the last good Clang.
+                  export ARTIFACT=clang-stage2-Rthinlto/latest
+                  # python3 llvm-zorg/zorg/jenkins/monorepo_build.py fetch
```
[the remark this answers is not in the document]
Comment (justice-adams-apple) [reply zorg/jenkins/jobs/jobs/lldb-cmake-intel:79]: No, good point, will update these to remove that stage

**Comment by justice-adams-apple** [comment]
@JDevlieghere thanks for the review 🙏 , updated based on the comments

**Review by JDevlieghere — APPROVED** [review]
Looks good to me

**Pull request merged**